### DISCUSSION TITLE
importing a map while paused does not indicate that the import is paused

### DISCUSSION THREAD

**Discussion by tovc** [ppy/osu#30388] [Ideas] [answered]
start a map, pause during gameplay, import a .osz file by dragging it into the window or by downloading a map via the beatmap listing, and the map will endlessly say 'beatmap import is initializing'. I'm aware that beatmap imports are paused during gameplay to avoid stuttering, but this should probably be communicated to the user better

![osu_2024-10-22_20-04-02](https://github.com/user-attachments/assets/7c04d124-dd4c-4f24-b97b-c63ca47db628)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/30402
